kafka-merge-pr.py

```diff
@@ -72,7 +72,7 @@
 
 DEV_BRANCH_NAME = "trunk"
 
-DEFAULT_FIX_VERSION = os.environ.get("DEFAULT_FIX_VERSION", "0.10.0.0")
+DEFAULT_FIX_VERSION = os.environ.get("DEFAULT_FIX_VERSION", "0.10.0.1")
```

Comment:
This one is a bit weird. Ideally, people would always run the merge script from trunk as it will include the latest changes to the script. I don't mind too much what we have here, so I'm OK with this change, but I thought I'd point it out.